src/elements/link-list/src/index.tsx

```diff
@@ -4,60 +4,88 @@ import * as React from 'react'
 import { jsx, Themed } from 'theme-ui'
 import { Glyph, Icon } from '@uswitch/trustyle.icon'
 
-export type Variant = 'base' | 'quickLinks'
+export type Variant = 'base' | 'quickLinks' | 'linkBlock'
 
 interface ListLinkProps extends React.HTMLAttributes<HTMLDivElement> {
   title?: string
   icon?: React.ReactNode
+  subtitle?: string
+  subtitleUrl?: string
   variant?: Variant
   className?: string
 }
 
+const VariantContext = React.createContext('base')
+
 const styles = (variant: Variant, element?: string) =>
   `elements.link-list.variants.${variant}${element ? `.${element}` : ''}`
 
 export const LinkList: React.FC<ListLinkProps> = ({
   children,
   title,
   icon,
+  subtitle,
+  subtitleUrl,
   variant = 'base',
   className
 }) => {
+  const IconComponent =
+    typeof icon === 'string' ? <Icon glyph={icon as Glyph} color="" /> : icon
+
   return (
-    <div className={className} sx={{ variant: styles(variant) }}>
-      {(title || icon) && (
-        <header
-          sx={{
-            display: 'flex',
-            alignItems: 'center',
-            variant: styles(variant, 'header')
-          }}
-        >
-          {icon}
-          {title && (
-            <Themed.h3
-              as="h2"
+    <VariantContext.Provider value={variant}>
```

Comment:
this is interesting, what do we use it for?

Reply:
just so the children of `LinkList` can know which variant is being used; specifically, so that the list items can be rendered without the top border when the variant is `linkBlock`: https://github.com/uswitch/trustyle/blob/21bed58cc5bc4bd629221aedbc5fafd22248eafe/src/elements/link-list/src/index.tsx#L120

Perhaps there's a better way of doing this ? I've looked at other components but didn't see this particular use case, where you need children to know about the parents variant.